src/main/java/seedu/address/model/BorrowerRecords.java

```diff
@@ -91,6 +91,35 @@ public Borrower getBorrowerFromId(BorrowerId id) throws NullPointerException {
         return borrowersMap.get(id);
     }
 
+    /**
+     * Returns true if the list contains an equivalent borrower as the given argument.
+     */
+    public boolean contains(Borrower toCheck) {
```

Comment:
Would you consider using the hashmap of borrowers to check if the borrower is inside instead? Might make the code cleaner

Reply:
This contains method is only used for checking inside the observable list when I use to replace a borrower in the list in the setBorrower methods. I think I'll just change the name of this method to avoid confusion.